org.opent2t.sample.thermostat.superpopular/com.nest.thermostat/js/constants.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * This file exports constants used by the Nest translators
+ */
+
+/** 
+ * Defines a new property on exports with the name, and value.
+ * This value is not writable, so acts as a true exported constant.
+ */
+function define(name, value) {
+    Object.defineProperty(exports, name, {
+        value: value,
+        enumerable: true,
+        writable: false,
+        configurable: false
+    });
+}
+
+define("SchemaMissingTemperature", 'Resource schema missing temperature units.');
```

Comment:
>define [](start = 0, length = 6)

Hmm so tests + source have to define these all over again? :( Beats the purpose of the consts then yes?  I was hoping these could be the same as the ones that the source uses